src/frontend/components/SøknadsSteg/VelgBarn/Barnekort/Barnekort.tsx

```diff
@@ -139,23 +135,38 @@ const Barnekort: React.FC<IBarnekortProps> = ({
                             ? 'hvilkebarn-utvidet.barn.søk-om.spm'
                             : 'hvilkebarn.barn.søk-om.spm',
                     })}
-                </StyledCheckbox>
-            </InformasjonsboksInnhold>
-            {erRegistrertManuelt && (
-                <FjernBarnKnapp barnId={barn.id} fjernBarnCallback={fjernBarnCallback} />
-            )}
-        </StyledBarnekort>
+                </Checkbox>
+                {erRegistrertManuelt && (
+                    <Button
+                        type={'button'}
+                        variant="tertiary"
+                        onClick={() => fjernBarnCallback(barn.id)}
+                        icon={<TrashFillIcon aria-hidden />}
+                    >
+                        <SpråkTekst id={'hvilkebarn.fjern-barn.knapp'} />
+                    </Button>
+                )}
+            </VStack>
+        </BarnekortContainer>
+    );
+};
+
+export const BarnekortContainer: React.FC<{ children?: ReactNode }> = ({ children }) => {
```

Comment:
Best practice for meg er å kun ha en komponent per fil, selv om jeg ser at det lå en `BarneKortInfo`-komponent nederst her fra før av. Jeg ville egentlig ha inlinet denne hvis den kun ble brukt ett sted, men jeg ser at den er brukt flere. Kan du legge den i en egen fil?